.github/actions/hugo-build-versioned-action/action.yaml

```diff
@@ -12,17 +12,28 @@ inputs:
     required: false
     description: Environment variable
     default: production
+  keep_master:
```

Comment:
Could we please standardize the typing here?

`keep_master` vs `fetch-from` below

Reply:
keep_master mimics the generate.sh, the fetch-from copies the rest of the actions. We should change all to kebab style